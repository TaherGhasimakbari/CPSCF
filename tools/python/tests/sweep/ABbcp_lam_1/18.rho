 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3854081749E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  3.200000002235E-01  6.799999997765E-01       0     1
  1.998439384127E-01 -1.998439382534E-01       1     2
  4.633071182962E-02 -4.633071178121E-02       2     2
  3.180923513066E-03 -3.180923512491E-03       3     2
 -1.763360221183E-03  1.763360222258E-03       4     2
 -7.233273434054E-04  7.233273495688E-04       5     2
 -9.942084380914E-05  9.942085057574E-05       6     2
  1.770057664111E-05 -1.770057473186E-05       7     2
  1.174864929307E-05 -1.174864999418E-05       8     2
  2.330674607055E-06 -2.330675372991E-06       9     2
 -7.810275979589E-08  7.810238461276E-08      10     2
 -1.772411625227E-07  1.772411078146E-07      11     2
 -4.793342938886E-08  4.793352890827E-08      12     2
 -2.544913412918E-09  2.545021420973E-09      13     2
  2.461153178867E-09 -2.461100988496E-09      14     2
  8.973327616565E-10 -8.973700739014E-10      15     2
  1.589809187075E-10 -1.590435283472E-10      16     1
