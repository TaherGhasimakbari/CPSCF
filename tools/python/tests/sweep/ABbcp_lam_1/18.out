 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.2000000022E-01    6.7999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3854081749E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.2339565276E+00
f_homo              
    2.2640000036E+00
pressure            
    3.6325315656E+00
mu_chain            
    5.8664880932E+00
stress              
   -4.4203325930E-11

STATISTICS          
N_star              
                  17
Final Error         
    4.4203325930E-09
Iterations          
                  10
Basis Time          
    2.6167106628E-01
SCF Time            
    8.8924884796E-02
